apps/src/lab2/views/ExtraLinksModal.tsx

```diff
@@ -338,6 +338,9 @@ const ProjectLinkData: React.FunctionComponent<ProjectLinkDataProps> = ({
   }
   const ownerInfo = projectLinkData.owner_info;
   const projectInfo = projectLinkData.project_info;
+  if (!ownerInfo || !projectInfo) {
```

Comment:
can you also update the type of `projectLinkData` to indicate these could be undefined? And we could also have a string `message`?

Reply:
Yes -definitely. Will do - thanks!